plugins/home-react/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@backstage/plugin-home-react",
+  "description": "A Backstage plugin that contains react components helps you build a home page",
+  "version": "0.0.1",
```

Comment:
You'll want to leave this as `0.0.0` and then it will get bumped when it gets release for the first time